targets/TARGET_Samsung/TARGET_SIDK_S5JS100/dcxo_update.cpp

```diff
@@ -0,0 +1,121 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include "mbed.h"
```

Comment:
this header file should not be part of target code, include what is required in this file

Reply:
The header is cleaned. https://github.com/ARMmbed/mbed-os/pull/12106/files#diff-c55655a694a274d0e0189bc1cf4fb021R4